packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html

```diff
@@ -1,57 +1,82 @@
                 data-ng-disabled="$ctrl.disabled"
-                title="{{$ctrl.getTitle()}}"
             >
-                <i class="oui-icon oui-icon-pen_line" aria-hidden="true"></i>
+                <span
+                    class="oui-icon oui-icon-pen_line oui-icon_small"
+                    aria-hidden="true"
+                ></span>
+                <span
+                    class="sr-only"
+                    data-translate="{{:: 'tuc_editableServiceName_change' | translate }}"
+                ></span>
             </button>
         </span>
+    </div>
 
-        <!-- IN EDITION -->
-        <span data-ng-if="$ctrl.inEdition" class="clearfix">
+    <div data-ng-if="$ctrl.inEdition && !$ctrl.loading.save">
+        <label
+            for="tuc_editableServiceName_input"
+            data-translate="tuc_editableServiceName_label"
+        ></label>
+        <div class="d-flex">
             <input
-                class="service-name-edit-input d-inline-block align-middle"
+                id="tuc_editableServiceName_input"
+                class="service-name-edit-input align-middle "
                 name="editableServiceNameInput"
                 data-ng-maxlength="$ctrl.maxlength"
                 data-ng-model="$ctrl.model.serviceName"
                 data-ng-readonly="$ctrl.loading.save"
                 type="text"
             />
 
-            <span class="edit-button-container d-inline-block align-middle">
-                <button
-                    type="submit"
-                    class="btn btn-editable"
-                    data-ng-disabled="$ctrl.loading.save || !editableServiceNameForm.editableServiceNameInput.$valid"
-                >
-                    <i class="ovh-font ovh-font-check"></i>
-                </button>
+            <button
+                type="submit"
+                class="btn btn-editable px-2 h-100"
+                data-ng-disabled="!editableServiceNameForm.editableServiceNameInput.$valid"
+            >
+                <span
+                    class="oui-icon oui-icon-success oui-icon_small"
+                    aria-hidden="true"
+                ></span>
+                <span
+                    class="sr-only"
+                    data-translate="{{:: 'tuc_editableServiceName_confirm' | translate }}"
+                ></span>
+            </button>
 
-                <button
-                    type="button"
-                    class="btn btn-editable"
-                    data-ng-click="$ctrl.cancelEdition()"
-                    data-ng-disabled="$ctrl.loading.save"
-                >
-                    <i class="ovh-font ovh-font-wrong"></i>
-                </button>
-            </span>
-        </span>
-    </h1>
-    <em class="align-middle" data-ng-bind="$ctrl.serviceName"></em>
+            <button
+                type="button"
+                class="btn btn-editable px-2 h-100"
+                data-ng-click=":: $ctrl.cancelEdition()"
```

Comment:
one time binding for ng-click?

Reply:
Yeah, I don't care about the return value so I'd better remove the watcher there